hack/common.sh

```diff
@@ -57,6 +60,7 @@ function compile() {
     -v controller_image=${controller_image} \
     -v webhook_image=${webhook_image} \
     -v build_init_image=${build_init_image} \
+    -v build_waiter_image=${build_waiter_image} \
```

Comment:
I wonder if we should just have one image for completion,build-init,waiting. (Steal a page from our lifecycle friends)

That obviously could happen in a different PR